<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dictionary</class>
 <widget class="QMainWindow" name="Dictionary">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dictionary</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="getDef">
      <property name="title">
       <string>Definitions</string>
      </property>
      <widget class="QLineEdit" name="getDefinitionStudentIDInput">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>74</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="getDefinitionButton">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>50</y>
         <width>131</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Display Definitions</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="definitionResultOutput">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>770</width>
         <height>98</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Student ID</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="completeModule">
      <property name="title">
       <string>Completed Modules</string>
      </property>
      <widget class="QLineEdit" name="studentIDforModule">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>30</y>
         <width>74</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="showModulesButton">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>52</y>
         <width>141</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Show Modules</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Student ID:</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="completedModulesList">
       <property name="geometry">
        <rect>
         <x>13</x>
         <y>83</y>
         <width>271</width>
         <height>81</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="RegisterStudents">
      <property name="title">
       <string>RegisterStudents</string>
      </property>
      <widget class="QLineEdit" name="studentIDinput">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>74</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>50</y>
         <width>31</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="markCompletedButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>121</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Mark Completed</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>291</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Completed Module # (Just type the number, ie: 1)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Student ID:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="markUncompletedButton">
       <property name="geometry">
        <rect>
         <x>159</x>
         <y>70</y>
         <width>151</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Mark Uncompleted</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="confirmOutput">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDictionary">
    <property name="title">
     <string>Dictionary</string>
    </property>
   </widget>
   <addaction name="menuDictionary"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
